test/change-stream.test.js

```diff
@@ -9,7 +9,8 @@ describe('PersistedModel.createChangeStream()', function() {
       var test = this;
       var app = loopback({ localRegistry: true });
       var ds = app.dataSource('ds', { connector: 'memory' });
-      this.Score = app.model('Score', {
+      var Score = app.registry.createModel('Score', { base: 'PersistedModel' });
```

Comment:
I believe `PersistedModel` is used as a base model by default, why is it necessary to specify it explicitly?